content/docs/learning-guide/tutorials/multiplayer/first-multiplayer-component.md

```diff
@@ -747,12 +749,22 @@ Now that we have created a data flow from the server to clients using network pr
     ```c++
     void MyFirstNetworkComponent::OnUpTimeChanged(double uptime)
     {
-        AZLOG("MyFirstNetworkComponent", "client = %f", uptime);
+        AZ_Printf("MyFirstNetworkComponent", "client = %f\n", uptime);
         static_cast<MyFirstNetworkComponentController*>(GetController())->SendConfirmUptime(uptime);
     }
     ```
 
-    ![Attaching Network Component to an Autonomous Entity](/images/user-guide/multiplayer/add_myfirstnetworkcomponent_to_player_prefab.png)
+1. Finally, add a check to `MyFirstNetworkComponentController::OnActivate` to ensure it is only connecting to the TickBus while it's executing on the server (e.g., is not the autonomous player client, which will also have a controller):
+    ```c++
+    void MyFirstNetworkComponentController::OnActivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
+    {
+        if (!IsAutonomous()) // add me
```

Comment:
did you want to leave the _//add me_ comment?

Reply:
What is the value of this comment?

Reply:
that's the check that we're asking them to add at this point - the rest of the code in this block should already exist. But if we think that's not helpful or needs to be worded better I'm all for changing it